drivers/Makefile.am

```diff
@@ -28,7 +28,7 @@ driverlibdir    = $(pkglibdir)
 driverdatadir   = $(pkgsysconfdir)
 driver_ldflags  = $(AM_LDFLAGS)
 driver_ldflags += -export-dynamic
-driver_ldflags += -export-symbols-regex libdrv_$(driver)_LTX_factory
```

Comment:
Nice catch!  Will fix this together with the other instance in a single commit attributing you.